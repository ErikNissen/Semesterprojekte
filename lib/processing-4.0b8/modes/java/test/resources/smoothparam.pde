version https://git-lfs.github.com/spec/v1
oid sha256:bb1b5be2ca5f5028da0b7d436af1b04779d27b25ec9f295fe23d46dd4f1015d1
size 174
