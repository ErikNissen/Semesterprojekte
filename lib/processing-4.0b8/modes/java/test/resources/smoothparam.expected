version https://git-lfs.github.com/spec/v1
oid sha256:14ac11fcd0e0fde1b8e4206c8b3516b911d9e66ff656a65eacf32309cfc618a9
size 961
